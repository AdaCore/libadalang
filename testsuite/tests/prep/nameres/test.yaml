driver: name-resolution
input_sources: [foo.adb]
preprocessor_data_file: prep.txt
preprocessor_path: ["d1", "d2"]
