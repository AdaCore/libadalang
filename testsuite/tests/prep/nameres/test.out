Analyzing foo.adb
#################

Resolving xrefs for node <CallStmt foo.adb:13:4-13:8>
*****************************************************

Expr: <Id "Bar" foo.adb:13:4-13:7>
  references:    <DefiningName "Bar" foo.adb:5:14-5:17>
  type:          None
  expected type: None


Done.
